src/patternfly/components/Card/card.scss

```diff
@@ -283,7 +284,7 @@
   @include pf-v6-mirror-inline-on-rtl;
 
   display: inline-block;
-  transition: transform var(--#{$card}__header-toggle-icon--TransitionDuration) var(--#{$card}__header-toggle-icon--TransitionTimingFunction);
+  transition: var(--#{$card}__header-toggle-icon--Transition); // TODO remove shorthand in breaking change
```

Comment:
Since `--#{$card}__header-toggle-icon--Transition` didn't exist before, you don't need to add it and can make it longhand.